File: Python/2942_find_words_containing_character/solution.py
def find_words_containing_char(words,find):
    result = list()
    for index,word in enumerate(words):
        if find in word:
            result.append(index)
    return result

words = ["abc","bcd","aaaa","cbc"]
find = "a"
print(find_words_containing_char(words,find))

File: JavaScript/2942_find_words_containing_character/solution.js
function find_words_containing_char(words,find) {
    let result = new Array();
    for (let [index,word] of words.entries()) {
        if (word.includes(find)) {
            result.push(index);
        }
    }
    return result;
}

const words = ["abc","bcd","aaaa","cbc"];
const find = "a";
console.log(find_words_containing_char(words,find));